tools/dailyjobs_query.py

```diff
@@ -66,14 +77,16 @@ def summarize(date, branch):
 
 
 def retrievedb(branch, date):
-    query = "select revision,count(result),sum(duration),platform from testjobs \
-             where branch='{0}' and date like '{1}%' and testtype not like '%build%' " \
-            "and testtype!='valgrind' group by revision,platform;"
-    query = query.format(branch, date)
-    print query
-    cur.execute(query)
-
-    for rows in cur.fetchall():
+    message = "select revision,count(result),sum(duration),platform from testjobs " \
+              " where branch='{0}' and date like '{1}%' and testtype not like '%build%' " \
+              "and testtype!='valgrind' group by revision,platform;"
+    data = session.query(Testjobs.revision, func.sum(Testjobs.duration), Testjobs.platform).\
+        filter(and_(Testjobs.branch == branch, Testjobs.date.like(date),
```

Comment:
here we are missing count(result), and 'and testtype not like '%build%' " \
 +              "and testtype!='valgrind' group by revision,platform;'